monitoring/instance_manager_collector.go

```diff
@@ -0,0 +1,174 @@
+package monitoring
+
+import (
+	"strings"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metricsclientset "k8s.io/metrics/pkg/client/clientset/versioned"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+	"github.com/longhorn/longhorn-manager/types"
+)
+
+type InstanceManagerCollector struct {
+	*baseCollector
+
+	kubeMetricsClient *metricsclientset.Clientset
+	namespace         string
+
+	instanceManagerCPUUsageMetric      metricInfo
```

Comment:
Remove the instanceManager prefix, these fields are internal, so they don't add additional value.
Since this is an InstanceManagerCollector :)

Reply:
it will also look nicer :)

`cpuUsageMetric` vs `CPUUsageMetric`

Reply:
Thank you! Fixed